lib/chemotion/chemicals_service.rb

```diff
@@ -135,11 +134,10 @@ def self.safety_section(product_link)
     def self.safety_phrases_merck(product_link)
       safety_section = safety_section(product_link)
       safety_array = safety_section.children.reject { |i| i.text.empty? }.map(&:text)
-      pictograms = safety_array[0].split(',')
-      verified_pictograms = pictograms.select { |pictogram| pictograms_hash.value?(pictogram) }
+      pictograms = safety_array[3].split(',')
```

Comment:
Index 3 will always exist?

Reply:
yes, we are doing data mining using web scraping. This is how the server usually responds.